app/views/reports/instrument_unavailable_reports/report_table.html.haml

```diff
@@ -0,0 +1,16 @@
+.updated_values{style: "display: none"}
```

Comment:
The partials are *very* close, but they are different in how they decide which columns are numeric (and get the `.currency` css class). The existing `report_table` partial makes assumptions about column indexes, where this one explicitly defines the numeric columns.

I'd rather use something like the `@numeric_columns` check for the existing reports too. Add a `TODO` or dive in now?